aict_tools/apply.py

```diff
@@ -106,9 +107,21 @@ def create_mask_h5py(
         name, (operator, value) = list(c.items())[0]
 
         before = mask.sum()
-        mask = np.logical_and(
-            mask, OPERATORS[operator](infile[key][name][start:end], value)
-        )
+        if isinstance(infile, tables.file.File):
```

Comment:
This should be it's own function.

Reply:
Ok
Actually need to check if this is still needed